to_multi_var

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ROOT_DIRECTORY="$( dirname "$( readlink -f "${BASH_SOURCE[0]}" )" )"
+readonly ROOT_DIRECTORY
```

Comment:
Why do we need this? Is it to prevent NEW files being created or something like this?